posydon/binary_evol/DT/step_detached.py

```diff
@@ -48,7 +48,8 @@
     "H-rich_Central_He_depleted",
     "H-rich_Core_C_burning",
     "H-rich_Central_C_depletion",
-    "H-rich_non_burning"]
+    "H-rich_non_burning",
+    "accreted_He_Core_H_burning"]
```

Comment:
Shouldn't we add this to the `LIST_ACCEPTABLE_STATES_FOR_HMS` instead of `LIST_ACCEPTABLE_STATES_FOR_postMS`, because the `Core_H_burning` should be MS.